docs/from_bosh_to_kube.md

```diff
@@ -357,13 +356,38 @@ tags:
 
 ## BPM
 
-### Entrypoint
+The BPM's are converted into containers of `ExtendedStatefulSet's` and `ExtendedJob's` pods. Each `process` in the BPM is run in a single `container` of kubernetes pods. Following subsections will describe about the mapping of BPM configuration into containers.
 
-### Environment
+### Entrypoint & Environment Variables
+
```

Comment:
These seem to be from https://bosh.io/docs/bpm/config/#process-schema

Reply:
Yes. I tried to map those to kube.